website/content/docs/auth/saml.mdx

```diff
@@ -0,0 +1,158 @@
+   Provider by either providing a URL for its Metadata document:
+
+   ```shell-session
+   $ vault write auth/saml/config \
+      default_role=admin \
+      idp_metadata_url=https://company.okta.com/app/abc123eb9xnIfzlaf697/sso/saml/metadata \
+      entity_id="https://my.vault/v1/auth/saml" \
+      acs_urls="https://my.vault/v1/auth/saml/callback"
+   ```
+
+   or by setting the configuration Metadata manually:
+
+   ```shell-session
+   $ vault write auth/saml/config \
+      default_role=admin \
+      idp_sso_url=https://company.okta.com/app/abc123eb9xnIfzlaf697/sso/saml \
+      idp_entity_id=https://www.okta.com/abc123eb9xnIfzlaf697 \
+      idp_cert=@path/to/cert.pem \
+      entity_id="https://my.vault/v1/auth/saml" \
+      acs_urls="https://my.vault/v1/auth/saml/callback"
+   ```
+
+1. Create a named role:
+
+   ```shell-session
+    $ vault write auth/saml/role/admin \
+       bound_subjects="*@hashicorp.com" \
+       bound_subjects_type="glob" \
+       token_policies="writer" \
+       bound_attributes=group="admin" \
+       ttl=1h
+   ```
+
+   This role authorizes users that have a subject with an `@hashicorp.com` suffix and
+   are in the `admin` group to authenticate. It also gives the resulting Vault token a
+   time-to-live of 1 hour and the `writer` policy.
+
+   For the complete list of configuration options, please see the
+   [API documentation](/vault/api-docs/auth/saml).
+
+### Assertion Consumer Service URLs
+
+The [`acs_urls`](/vault/api-docs/auth/saml#acs_urls) configuration parameter determines
+where the SAML Response will be sent after users authenticate with the configured SAML
+Identity Provider in their browser.
+
+The values provided to Vault must:
+
+- Match or be a subset of the configured values for the SAML application within the
+  configured Identity Provider.
+- Be directed to the auth method's [Assertion Consumer Service
+  Callback](/vault/api-docs/auth/saml#assertion-consumer-service-callback) API.
+
+<Note>
+  It is highly recommended and enforced by some identity providers to TLS-protect the
+  Assertion Consumer Service URLs. A warning will be returned from Vault if any of the
+  configured Assertion Consumer Service URLs are not protected by TLS.
+</Note>
+
+To support a single auth method mount being used across Vault [replication](/vault/docs/enterprise/replication)
+clusters, `acs_urls` supports configuration of multiple values. For example, to support
+SAML authentication on a primary and secondary Vault cluster, the following `acs_urls`
+configuration could be given:
+
+   ```shell-session
+   $ vault write auth/saml/config \
+      acs_urls="https://primary.vault/v1/auth/saml/callback,https://secondary.vault/v1/auth/saml/callback"
+   ```
+
+The Vault UI and CLI will automatically request the proper Assertion Consumer Service URL
+for the cluster they're configured to communicate with. This means that the entirety of the
+authentication flow will stay within the local cluster.
+
```

Comment:
+ add note on how to configure the ACS for auth methods within namespaces.

Reply:
Done in https://github.com/hashicorp/vault/pull/23183/commits/b66428d63f126a4a9d64050c209e44688ded2d9c.